game-app/game-core/src/test/java/games/strategy/triplea/ui/panels/map/MapRouteDrawerTest.java

```diff
@@ -138,14 +138,10 @@ void verifySequenceIsTrulyMonotonic(final Point2D[] points) {
             Arguments.of(new Double(-1, -1)),
             Arguments.of(new Double(1, -1), new Double(1, -1), new Double(1, -1)),
             Arguments.of(new Double(-1, -1), new Double(1, 1), new Double(0, 0)),
-            // 13.44 is chosen specifically to test a rare edge case that used to cause an exception
-            // in the past. We used to add a minimal positive double to indices to make them
-            // strictly increasing as required by commons-math. However in case of 13.44 the
-            // expression 13.44 + Double.MIN_NORMAL == 13.44 evaluates to true, so the sequence is
-            // no longer strictly increasing. So we use the square of this value (because the code
-            // computes the square root) in order to verify we don't get this issue again.
-            Arguments.of(
-                new Double(0, 0), new Double(0, 13.44 * 13.44), new Double(0, 13.44 * 13.44)))
+            // We used to add a minimal positive double to indices to make them strictly increasing
```

Comment:
At this point, is the comment even really needed? It does not really matter what we 'used to do', someone reading this test case would only want to know why the test case is interesting and why we expect it to pass. Noting that the value was added for an 'edge' case I wonder if it's just unnecessary context at this point.